sonar-go-to-slang/build.gradle.kts

```diff
@@ -62,7 +62,7 @@ if (isCi) {
         callMake("build")
     }
 
-    tasks.register<Exec>("lintGoCode") {
+    val lintGoCode = tasks.register<Exec>("lintGoCode") {
```

Comment:
Maybe the `goLangCiLint` or similar will be a better name is it executes golangci-lint?

Reply:
I copied it from sonar-iac, but you are probably right and it should be renamed there as well